src/Features/Core/Portable/CodeRefactorings/CodeRefactoringContextExtensions.cs

```diff
@@ -30,29 +29,28 @@ internal static class CodeRefactoringContextExtensions
             }
         }
 
-        internal static async Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(this CodeRefactoringContext context)
+        internal static Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(this CodeRefactoringContext context)
             where TSyntaxNode : SyntaxNode
-        {
-            (var document, var span, var cancellationToken) = context;
-            var potentialNodes = await GetRelevantNodes<TSyntaxNode>(document, span, cancellationToken).ConfigureAwait(false);
-            return potentialNodes.FirstOrDefault();
-        }
+            => TryGetRelevantNodeAsync<TSyntaxNode>(context.Document, context.Span, context.CancellationToken);
 
-        internal static async Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(this Document document, TextSpan span, Func<TSyntaxNode, bool> predicate, CancellationToken cancellationToken)
-where TSyntaxNode : SyntaxNode
-        {
-            var potentialNodes = await GetRelevantNodes<TSyntaxNode>(document, span, cancellationToken).ConfigureAwait(false);
-            return potentialNodes.FirstOrDefault(predicate);
-        }
+        internal static Task<ImmutableArray<TSyntaxNode>> TryGetSelectedNodesAsync<TSyntaxNode>(this CodeRefactoringContext context)
+            where TSyntaxNode : SyntaxNode
+            => TryGetRelevantNodesAsync<TSyntaxNode>(context.Document, context.Span, context.CancellationToken);
 
-        internal static async Task<TSyntaxNode> TryGetSelectedNodeAsync<TSyntaxNode>(this Document document, TextSpan span, CancellationToken cancellationToken)
-where TSyntaxNode : SyntaxNode
+        internal static async Task<TSyntaxNode> TryGetRelevantNodeAsync<TSyntaxNode>(
+            this Document document,
+            TextSpan span,
+            CancellationToken cancellationToken)
+            where TSyntaxNode : SyntaxNode
         {
-            var potentialNodes = await GetRelevantNodes<TSyntaxNode>(document, span, cancellationToken).ConfigureAwait(false);
+            var potentialNodes = await TryGetRelevantNodesAsync<TSyntaxNode>(document, span, cancellationToken).ConfigureAwait(false);
             return potentialNodes.FirstOrDefault();
         }
 
-        private static async Task<ImmutableArray<TSyntaxNode>> GetRelevantNodes<TSyntaxNode>(Document document, TextSpan span, CancellationToken cancellationToken) where TSyntaxNode : SyntaxNode
+        internal static async Task<ImmutableArray<TSyntaxNode>> TryGetRelevantNodesAsync<TSyntaxNode>(
+            this Document document,
+            TextSpan span,
+            CancellationToken cancellationToken) where TSyntaxNode : SyntaxNode
```

Comment:
let's be consistent either calling it "TryGetRelevant" or "TryGetSelected".  (unless there is a super ood reason for both names...)